web/client/utils/cesium/GeoServerBILTerrainProvider.js

```diff
@@ -302,13 +302,14 @@ function parseOptions(options) {
             severUrl = severUrl.substring(0, index);
         }
         const urlGetCapabilities = `${severUrl}?SERVICE=WMS&REQUEST=GetCapabilities&tiled=true`;
-        let updatedCapabilitiesUrl = getCapabilitiesUrl({ url: urlGetCapabilities, name: options.layerName });
-        if (defined(options.proxy)) {
-            updatedCapabilitiesUrl = options.proxy.getURL(updatedCapabilitiesUrl);
-        }
+        let updatedCapabilitiesUrl = getCapabilitiesUrl({ url: urlGetCapabilities, name: options.layerName, authKey: options.authkey });
```

Comment:
opions.authkey does not exist so we need to remove it
```suggestion
        let updatedCapabilitiesUrl = getCapabilitiesUrl({ url: urlGetCapabilities, name: options.layerName });
```